src/theme.js

```diff
@@ -239,6 +244,21 @@ const typography = {
 };
 
 
+const table = {
+  primary: {
+    borderBottom: `1px solid ${colors.greyPalette.lighter}`,
```

Comment:
This was what I was talking about btw

Reply:
yeah, I would add a prop in `<Table>`. I would make it `borderColor` so you can always have `1px solid {borderColor}`